src/index.html

```diff
@@ -5,8 +5,316 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header__logo" href="#home">
+        <img
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__ul">
+          <li class="nav__li-item">
+            <a class="nav__link is-active" href="#Apple">Apple</a>
+          </li>
+          <li class="nav__li-item">
+            <a class="nav__link" href="#Sumsung">Samsung</a>
+          </li>
+          <li class="nav__li-item">
+            <a class="nav__link" href="#Смартфоны и Телефоны">Смартфоны и Телефоны</a>
+          </li>
+          <li class="nav__li-item" data-qa="nav-hover">
+            <a class="nav__link" href="#Ноутбуки и Компьютеры">Ноутбуки и Компьютеры</a>
+          </li>
+          <li class="nav__li-item">
+            <a class="nav__link" href="#Гаджеты"> Гаджеты </a>
+          </li>
+          <li class="nav__li-item">
+            <a class="nav__link" href="#Планшеты">Планшеты</a>
+          </li>
+          <li class="nav__li-item">
+            <a class="nav__link" href="#Фото">Фото</a>
+          </li>
+          <li class="nav__li-item">
+            <a class="nav__link" href="#Видео">Видео</a>
+          </li>
+
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="container">
+        <div class="container__card card" data-qa="card">
+          <img
+            class="card__logo"
+            src="/images/imac.jpeg"
+            alt="mac_photo">
+    
+          <h1 class="card__product-name">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </h1>
+    
+          <p class="card__product-code">
+            Код товара: 195434
+          </p>
+    
+          <div class="stars stars--4">
+            <div class="stars__star star"></div>
+            <div class="stars__star star"></div>
+            <div class="stars__star star"></div>
+            <div class="stars__star star"></div>
+            <div class="stars__star star"></div>
+            <p class="stars__review">Отзывов: 5</p>
+          </div>
+    
+          <h2 class="card__price-block">
+            Цена:
+            <p class="price">69 999 грн</p>
+          </h2>
```

Comment:
https://stackoverflow.com/questions/19779519/is-it-valid-to-have-paragraph-elements-inside-of-a-heading-tag-in-html5-p-insid